src/Tasks/NativeMethods.cs

```diff
@@ -813,7 +813,7 @@ internal static bool MakeHardLink(string newFileName, string exitingFileName, re
             else
             {
                 hardLinkCreated = link(exitingFileName, newFileName) == 0;
-                errorMessage = hardLinkCreated ? null : "The link() library call failed with the following error code: " + Marshal.GetLastWin32Error();
+                errorMessage = hardLinkCreated ? null : log.GetResourceMessage("Copy.LinklibraryFailedPrefix") + Marshal.GetLastWin32Error();
```

Comment:
This same line also appears in MakeSymbolicLink; you should change that one, too.

Reply:
Done